projects/grpc-swift/llvmsymbol.diff

```diff
@@ -0,0 +1,51 @@
+diff --git a/llvm/lib/DebugInfo/Symbolize/CMakeLists.txt b/llvm/lib/DebugInfo/Symbolize/CMakeLists.txt
+index acfb3bd0e..5c4cf9763 100644
+--- a/llvm/lib/DebugInfo/Symbolize/CMakeLists.txt
++++ b/llvm/lib/DebugInfo/Symbolize/CMakeLists.txt
+@@ -12,4 +12,12 @@ add_llvm_component_library(LLVMSymbolize
+   Object
+   Support
+   Demangle
+-  )
++
++  LINK_LIBS
++  /usr/lib/swift_static/linux/libswiftCore.a
++  /usr/lib/swift_static/linux/libswiftImageInspectionShared.a
++  /usr/lib/swift_static/linux/libicui18nswift.a
++  /usr/lib/swift_static/linux/libicuucswift.a
++  /usr/lib/swift_static/linux/libicudataswift.a
++  /usr/lib/x86_64-linux-gnu/libstdc++.so.6
++)
+diff --git a/llvm/lib/DebugInfo/Symbolize/Symbolize.cpp b/llvm/lib/DebugInfo/Symbolize/Symbolize.cpp
+index 4c3f3a3767e1..aa7b9f0f5abb 100644
+--- a/llvm/lib/DebugInfo/Symbolize/Symbolize.cpp
++++ b/llvm/lib/DebugInfo/Symbolize/Symbolize.cpp
+@@ -36,6 +36,13 @@
+ #include <cassert>
+ #include <cstring>
+ 
++
++extern "C" char *swift_demangle(const char *mangledName,
```

Comment:
We probably discussed this before, but is this something we could get upstreamed (or fix it in the sanitizers?)

Reply:
Indeed we discussed this.
If I remember correctly, there were multiple steps.
The first one was to get something generic for swift projects.

And about this, there was the question if we should grow the base-builder image by adding another 1G toolchain, or if we should have a base-builder-swift image.
@jonathanmetzman do I remember correctly ? Do you want me to propose a PR to go one way or another ?

Reply:
@oliverchang I think we should probably just wait for @asraa 's work on splitting up base-builder to add something generic.
What do you think?

Reply:
Yes, let's wait for the image split before we do this.